services/wfm/src/main/java/org/openkilda/wfm/topology/flow/bolts/CrudBolt.java

```diff
@@ -581,19 +582,22 @@ private void handleCreateRequest(CommandMessage message, Tuple tuple) throws IOE
         Flow requestedFlow = ((FlowCreateRequest) message.getData()).getPayload();
 
         FlowPair<PathInfoData, PathInfoData> path;
+        final String errorType = "Could not create flow";
         try {
             flowValidator.validate(requestedFlow);
 
             path = pathComputer.getPath(requestedFlow, Strategy.COST);
             logger.info("Creating flow {}. Found path: {}, correlationId: {}", requestedFlow.getFlowId(), path,
                     message.getCorrelationId());
-
         } catch (FlowValidationException e) {
             throw new MessageException(message.getCorrelationId(), System.currentTimeMillis(),
-                    ErrorType.ALREADY_EXISTS, "Could not create flow", e.getMessage());
+                    ErrorType.ALREADY_EXISTS, errorType, e.getMessage());
```

Comment:
Can ErrorType be a part of `ValidationException`?

PS ALREADY_EXIST looks not always valid for `FlowValidationException`.